src/schnetpack/representation/painn.py

```diff
@@ -223,7 +254,27 @@ def forward(self, inputs: Dict[str, torch.Tensor]):
         else:
             filter_list = torch.split(filters, 3 * self.n_atom_basis, dim=-1)
 
-        q = self.embedding(atomic_numbers)[:, None]
+
+        if self.activate_charge_spin_embedding:
+            # get inputs for spin and charge embedding 
+            # (to avoid error not having total charge /spin multiplicity in db if embedding not used)
+            total_charge = inputs[properties.total_charge]
+            spin = inputs[properties.spin_multiplicity]
+            # specific nuclear embeeding with electron configuration
+            z = self.nuclear_embedding(atomic_numbers)
+
+            # specific total charge embeeding
+            c = self.charge_embedding(x = z, E = total_charge, num_batch = num_batch, batch_seg = batch_seg)
+
+            # specific spin embeeding
+            s = self.magmom_embedding(x = z, E = spin, num_batch = num_batch, batch_seg = batch_seg)
+
+            # additive combining nuclear, charge and spin embeeding
+            q = (z + c + s)[:,None] 
+
+        else:
+            q = self.embedding(atomic_numbers)[:, None]
```

Comment:
I dont get why we need the `[:, None]` only in one case. Would it be possible to remove the `else` statement and only run the `if` code.

e.g.
```
    embedding = self.embedding(atomic_numbers)[:, None]
    if self.activate_charge_spin_embedding:
            # get inputs for spin and charge embedding 
            # (to avoid error not having total charge /spin multiplicity in db if embedding not used)
            total_charge = inputs[properties.total_charge]
            spin = inputs[properties.spin_multiplicity]          

            # specific total charge embeeding
            c = self.charge_embedding(x = embedding, E = total_charge, num_batch = num_batch, batch_seg = batch_seg)

            # specific spin embeeding
            s = self.magmom_embedding(x = z, E = spin, num_batch = num_batch, batch_seg = batch_seg)

            # additive combining nuclear, charge and spin embeeding
            embedding = (embedding + c + s)[:,None] (?)
 ```

Reply:
and I suggest to change the names from `c` and `s` to `charge_embedding` and `spin_embedding`

Reply:
done in 2156399